doc/source/examples/diffraction_objects_example.rst

```diff
@@ -145,15 +145,38 @@ DiffractionObject convenience functions
 
 will return ``True``.
 
-3) make arithmetic operations on the intensities of diffraction objects. e.g.,
+3. make arithmetic operations on the intensities of diffraction objects.
+For example, you can do scalar operations on a single diffraction object,
+which will modify the intensity values (``yarrays``) without affecting other properties:
 
 .. code-block:: python
 
-    doubled_object = 2 * diff_object1     # Double the intensities
-    sum_object = diff_object1 + diff_object2    # Sum the intensities
-    subtract_scaled = diff_object1 - 5 * diff_object2   # subtract 5 * obj2 from obj 1
+    increased_intensity = diff_object1 + 5      # Increases the intensities by 5
+    decreased_intensity = diff_object1 - 1      # Decreases the intensities by 1
+    doubled_object = 2 * diff_object1           # Double the intensities
+    reduced_intensity = diff_object1 / 2        # Halves the intensities
 
-4) get the value of the DiffractionObject at a given point in one of the xarrays
+You can also do binary operations between two diffraction objects, as long as their yarrays have the same length.
```

Comment:
is the condition "same length xarray" or "on the same `q/tth/d-array`"?  I think it should be the latter or we are adding things incorrectly.  Please can you check the code.  If the code is doing it right, change the doc.  If hte doc is right, we probably should revisit hte tests and code.

Reply:
Yeah I think the code only checks for the length of yarrays. I can edit the test on this PR.

Reply:
Actually I wanna talk to @bobleesj first before making any changes. His test cases seem to be carefully chosen.. Made this into a new issue #298. @sbillinge I changed the docs and it's ready for review